### PR TITLE
feat(sdk): Add missing code

### PR DESCRIPTION
**Pull request by KevinDavilaDotCMS**
### Proposed Changes
* Added CustomNoComponent code (missed in some PR)
* Added env variable for HTTPS pages

### CODE REVIEW

File: examples/nextjs/.env.local.example [hunk 1]
```diff
@@ -1,4 +1,6 @@
 DOTCMS_AUTH_TOKEN=
 NEXT_PUBLIC_DOTCMS_HOST=http://localhost:8080
 NEXT_PUBLIC_EXPERIMENTS_API_KEY=analytic-api-key-from-dotcms-portlet
-NEXT_PUBLIC_EXPERIMENTS_DEBUG=true
\ No newline at end of file
+NEXT_PUBLIC_EXPERIMENTS_DEBUG=true
+# This is for HTTPS server
```
Comment (zJaaal) [inline examples/nextjs/.env.local.example:5]: Can we add a link with the docs of node for a further explanation of how this works?

**Review by zJaaal — APPROVED** [review]
Approved with one suggestion

File: examples/nextjs/.env.local.example (hunk 1 above)
Comment (fmontes) [inline examples/nextjs/.env.local.example:5]:
```suggestion
# If your local dotcms instance is running in https, this setting allows Node.js to connect to servers with invalid SSL certificates. For testing purposes only.
```

**Review by fmontes — CHANGES_REQUESTED** [review]

File: examples/nextjs/.env.local.example (hunk 1 above)
Comment (KevinDavilaDotCMS) [reply examples/nextjs/.env.local.example]: Added!

**Review by nicobytes — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**